src/draw/ElkLayout.js

```diff
@@ -301,6 +305,8 @@ class ElkLayout extends DefaultLayout {
     const currentDepth = parentNode.depth + 1;
 
     return allLinks
+    // We ignore links for nodes that we don't know about.
```

Comment:
`Ignore links that's not related to given container`

Reply:
done

Reply:
done